include/net/openthread.h

```diff
@@ -17,27 +28,80 @@
 extern "C" {
 #endif
 
+/**
+ * @cond INTERNAL_HIDDEN
+ */
+/**
+ * @brief Type of pkt_list
+ */
 struct pkt_list_elem {
 	struct net_pkt *pkt;
 };
 
+/**
+ * @brief OpenThread l2 private data.
+ */
 struct openthread_context {
+	/** Pointer to OpenThread stack instance */
 	otInstance *instance;
+
+	/** Pointer to OpenThread network interface */
 	struct net_if *iface;
+
+	/** Index indicates the head of pkt_list ring buffer */
 	uint16_t pkt_list_in_idx;
+
+	/** Index indicates the tail of pkt_list ring buffer */
 	uint16_t pkt_list_out_idx;
+
+	/** Flag indicates that pkt_list is full */
 	uint8_t pkt_list_full;
+
+	/** Array for storing net_pkt for OpenThread internal usage */
 	struct pkt_list_elem pkt_list[CONFIG_OPENTHREAD_PKT_LIST_SIZE];
 };
+/**
+ * INTERNAL_HIDDEN @endcond
```

Comment:
I did not know that this works too, usually I have just placed `/** @endcond */` here

Reply:
It's quite common for instance in `kernel.h`